Userland/Libraries/LibWeb/HTML/Scripting/ExceptionReporter.cpp

```diff
@@ -36,16 +36,18 @@ void print_error_from_value(JS::Value const& value, ErrorInPromise error_in_prom
             }
         }
     } else {
-        dbgln("\033[31;1mUnhandled JavaScript exception:\033[0m {}", value);
+        dbgln("\033[31;1mUnhandled JavaScript exception{}:\033[0m {}", error_in_promise == ErrorInPromise::Yes ? " (in promise)" : "", value);
     }
 
     print_error_to_console(value);
 }
 
 void print_error_to_console(JS::Value const& value)
 {
-    auto& current_realm = *Bindings::main_thread_vm().current_realm();
-    auto& console = current_realm.intrinsics().console_object()->console();
+    if (!value.is_object())
+        return;
```

Comment:
An error can be a non-object for example try
`throw 3;` (Not something you should do is js but something you can do)
The message for this doesn't need to have detail since we don't have any detail here.
But dropping them silently would be confusing.

(If this already prints something else than it's fine)

Reply:
Nice!